### DISCUSSION TITLE
List columns

### DISCUSSION THREAD

**Discussion by clawes** [filamentphp/filament#465] [Help]
How can I control which columns are shown in the list view?
I don't want all the fields in a model  to be listed by default.

**Comment by danharrin** [comment]
The only columns in the list view are the ones in the `table()` method. The form fields are in `form()`.

**Comment by clawes** [comment]
Thanks!  I figured it out after reading the documentation for a second time.

On Fri, Jun 25, 2021 at 11:09 AM Dan Harrin ***@***.***>
wrote:

> The only columns in the list view are the ones in the table() method. The
> form fields are in form().
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/laravel-filament/filament/discussions/465#discussioncomment-921261>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABOX6RZGBCRSPHDQKPW3433TUSLZVANCNFSM47KBJEGQ>
> .
>


-- 
Regards,
Conrad Lawes
